### PR TITLE
clean build-roofs and add lldb option

### PR DESCRIPTION
**Pull request by seanshpark**
- clean unused __MachineTriple
- add indents to case
- add lld 3.6 or 3.8 as option

this will solve #7085 also.

### CODE REVIEW

**Comment by seanshpark** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please

**Comment by seanshpark** [comment]
@jkotas @lucenticus PTAL

File: cross/build-rootfs.sh [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 usage()
 {
-    echo "Usage: $0 [BuildArch] [UbuntuCodeName]"
-    echo "BuildArch can be: arm, arm-softfp, arm64"
+    echo "Usage: $0 [BuildArch] [UbuntuCodeName] [lldbx.y]"
+    echo "BuildArch can be: arm(default), arm-softfp, arm64"
     echo "UbuntuCodeName - optional, Code name for Ubuntu, can be: trusty(default), vivid, wily, xenial. If BuildArch is arm-softfp, UbuntuCodeName is ignored."
+    ccho "lldbx.y - optional, LLDB version, can be: lldb3.6(default), lldb3.8"
```
Comment (lucenticus) [inline cross/build-rootfs.sh]: `ccho -> echo`

Comment (seanshpark) [inline cross/build-rootfs.sh]: thanks!

**Comment by seanshpark** [comment]
CC @qmfrederik

**Pull request merged**